protocol-definitions/ReplicatedMap.yaml

```diff
@@ -852,3 +852,75 @@ methods:
           doc: |
             partition id of the entries
     response: {}
+  - id: 20
+    name: fetchEntryViews
+    since: 2.7
+    doc: |
+      Allows iteration over ReplicatedMapEntryViewHolder objects. The changes happened during the iteration may not be included in the iterated EntryViews. This method
+      will throw an exception if there is no replicated record store with the given replicated map name and partition id. This method will consume some memory in the 
+      member with the default timeout of 300 seconds that is reset after each fetch. Sending endEntryViewIteration or timing out will release the resource.
+    request:
+      retryable: true
```

Comment:
You say that this API causes some memory allocation and keeps it at server memory. What happens to that memory on retry since you made it retryable?

Reply:
The memory per client iteration is constant.
If a request is retried with same cursor id it doesn't allocate new memory.